system/topic_state_monitor/launch/topic_state_monitor.launch.xml

```diff
@@ -19,6 +20,7 @@
     <param name="warn_rate" value="$(var warn_rate)" />
     <param name="error_rate" value="$(var error_rate)" />
     <param name="timeout" value="$(var timeout)" />
+    <param name="transient_local" value="$(var transient_local)" />
```

Comment:
Please fix here as well :pray:

Reply:
Sorry. I overlooked this. Fixed at 9c0ca72.